README.md

```diff
@@ -1,7 +1,35 @@
-### Code for Paper: “Low-Resource” Text Classification: A Parameter-Free Classification Method with Compressors
+# Code for Paper: “Low-Resource” Text Classification: A Parameter-Free Classification Method with Compressors
 
 This paper is accepted to Findings of [ACL2023](https://aclanthology.org/2023.findings-acl.426/).
 
+## Getting Started
+
+This codebase is [available on pypi.org via](https://pypi.org/project/npc-gzip) 
+
+
+```bash
+
+pip install npc-gzip
+
+```
+
+See the [examples](./examples/imdb.py) directory for example usage.
+
+
+## Testing
+
+This package utilizes `poetry` to maintain its dependencies and `pytest` to execute tests. To get started running the tests:
+
+```bash
+
+poetry shell
+poetry install
+pytest
+
+```
+
+
+
 ### Require
```

Comment:
It wouldn't be obvious to someone unfamiliar with the code that this step and below are for using the original codebase. I suggest putting something like this above it:

```markdown
### Original codebase

The code from before it was reorganized and packaged is available in the [`original_codebase/`](original_codebase/) directory. The following instructions apply to that version of the code.
```

And then changing all the `h3` (`###`) headings below it, including the `### Require` heading, to `h4` (`####`).

If decide to make such a change, please feel free to use that wording (or other wording). If you like, I can open a PR against the `v0.0.1-packaging` branch (like I did in #28), but I would only do that if it would be more convenient--I am *not* requesting that the change be made in that way.